### PR TITLE
Git Push constraint with E2E Test + Fix broken tests

### PR DESCRIPTION
**Pull request by thala321**
Video demo: https://www.loom.com/share/2e53db71969f4ea8a4f2f2066ecea361

**Our current CI/CD**
image

---
**Zoom in image:**
image

**Features:**
- Add more test for solana + fix broken ones (**done**)
image
- Pooling DB data (**done**)
- Delete migration table when create a new index (**done**)
- Health check will run before all other test  (**done**)
image
- A Makefile command (detail is in the README.md) so developers can hook their git push to E2E Tests  (**done**)
- A Makefile command (detail is in the README.md) so developers can un-hook their git push (**done**)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,8 +42,11 @@ index-manager/generated/*
 
 migrations/
 e2e-test/substrate/report.html
-
 e2e-test/substrate/output.xml
-
 e2e-test/substrate/log.html
+log.html
```
Comment (anhhuy0501) [inline .gitignore:47]: Choose output folder for report file

File: e2e-test/substrate/substrate.robot [hunk 2]
```diff
@@ -25,20 +25,20 @@ Check index-manager is up
 ##############
```
Comment (anhhuy0501) [inline e2e-test/substrate/substrate.robot:25]: add solana test

**Comment by thala321** [comment]
- Add more test for solana
image
- Pooling DB data
- Delete migration table when create a new index

**Review by thala321 — COMMENTED** [review]
We should squash merge this PR. There are too many unnecessary commits

**Comment by thala321** [comment]
E2E Test duration in the local will be around 10 -> 30 mins
@daywednes @anhhuy0501

**Review by anhhuy0501 — APPROVED** [review]
LGTM

**Pull request merged**